src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/troubleshooting/missing-otlp-data.mdx

```diff
@@ -0,0 +1,57 @@
+---
+title: "Troubleshoot OpenTelemetry: Missing OTLP data"
+tags:
+  - Integrations
+  - Open source telemetry integrations
+  - OpenTelemetry
+  - Troubleshooting
+metaDescription: Troubleshoot tips for when you use OpenTelemetry with New Relic
+freshnessValidatedDate: 2024-05-01
+---
+
+import moreintegrationsOtlpFacetQuery from 'images/more-integrations_screenshot-crop_otlp-facet-query.webp'
+
+## Problem
+
+You sent OpenTelemetry metrics, logs, or traces using OTLP, but you can't find the data in New Relic. 
+
+## Solution
+
+Before digging deeper, make sure you've checked the following:
+
+* The OTLP endpoint configured matches one of our [documented endpoints](/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/get-started/opentelemetry-set-up-your-app/#review-settings), is properly formatted, and includes the [default port for OTLP/gRPC](https://github.com/open-telemetry/opentelemetry-specification/blob/v1.8.0/specification/protocol/otlp.md#otlpgrpc-default-port), 4317, or the [default port for OTLP/HTTP](https://github.com/open-telemetry/opentelemetry-specification/blob/v1.8.0/specification/protocol/otlp.md#otlphttp-default-port), 4318. Port 443 is also supported for either transport. Please note the specific endpoint for FedRAMP compliance, if applicable.
+* Confirm that you're using the correct endpoint based on your region. For example, if you're based in Europe and configure your exporter to send data to our US endpoint, data will fail to be exported because the endpoints are region-specific.
+* The outbound traffic is not restricted by a firewall. Our [networks document](/docs/using-new-relic/cross-product-functions/install-configure/networks/#opentelemetry) explains which domains and network blocks you may need to explicitly allow.
+* The client is configured to use TLS 1.2 or higher and the request includes the `api-key` header with a valid New Relic <InlinePopover type="licenseKey" />.
+* Requests include valid protobuf payloads and use gRPC or HTTP transport, preferably with [gzip compression](/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-concepts/#compression) enabled. Request payloads should be no larger than 1MB (10^6 bytes). [JSON-encoded protobuf](https://github.com/open-telemetry/opentelemetry-specification/blob/v1.8.0/specification/protocol/otlp.md#otlphttp-connection) payloads are also supported.
+* You might be receiving `4xx` or `5xx` response codes. Client output and logs do not show error response codes.
+
+If you've checked the above, try out these New Relic features:
+
+* Check your data management hub to [facet data ingest](/docs/data-apis/manage-data/manage-data-coming-new-relic/#facet-data-ingest) and determine how much data is arriving from various sources. 
```

Comment:
What are you facet-ing with in the data management hub? If the recommendation is `instrumentation.provider` or `newrelic.source`, then I should delete this bullet point (and depend on the second bullet point)